namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-contacts-dev/resources/versions.tf

```diff
@@ -0,0 +1,21 @@
+terraform {
+  required_version = ">= 1.2.5"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.67.0"
+    }
+    github = {
+      source  = "integrations/github"
+      version = "~> 5.39.0"
+    }
+    kubernetes = {
+      source  = "hashicorp/kubernetes"
+      version = "~> 2.23.0"
+    }
+    random = {
+      source  = "hashicorp/random"
+      version = "~> 3.6.0"
+    }
```

Comment:
Maybe lose the random provider if not used?

Reply:
not sure if its not used, by looking at the terraform/random provider wiki says 'Supports the use of randomness within Terraform configurations. This is a logical provider, which means that it works entirely within Terraform logic, and does not interact with any other services.' so I guess its ok to leave it for now, most of the other configurations have this too

Reply:
It is still failing the tflint because of this.

Reply:
Warning: namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-contacts-dev/resources/versions.tf:16:5: Warning - provider 'random' is declared in required_providers but not used by the module (terraform_unused_required_providers)
Error: Process completed with exit code 2.

Reply:
ahha, :) removed that now